packages/mixer/src/components/Withdraw/WithdrawingModal.tsx

```diff
@@ -66,81 +116,70 @@ const WithdrawingModal: React.FC<WithdrawingModalProps> = ({ canCancel, cancel,
     }
 
     const address = withdrawTxInfo.account;
-    return `${address.slice(0, 20)}...${address.slice(address.length - 10, address.length)}`;
+    return `${address.slice(0, 6)}...${address.slice(address.length - 6, address.length)}`;
   }, [withdrawTxInfo]);
 
   const mixerEntries = useMixerGroupsEntries();
   const selectedMixerItem = mixerEntries.getItemsOf().find(matchedMixer => matchedMixer.id === withdrawTxInfo?.note.id);
 
   return (
-    <Dialog open={stage > WithdrawState.Ideal} fullWidth maxWidth={'sm'}>
```

Comment:
- [ ] Implement this logic in Withdraw.tsx for showing Modal so that it closes on success